backend/jobs/eventsRetriever.ts

```diff
@@ -15,20 +15,20 @@ export default class EventsRetriever {
     this.whereaboutsApi = whereaboutsApi
   }
 
-  async retrieveEventsForDay(day: Moment): Promise<string[][]> {
+  async retrieveEventsForPastMonth(date: Moment): Promise<string[][]> {
     const tokens = await this.tokenSource.getTokens()
     const parser = parse({
       delimiter: ',',
       // line 1 contains the column headers. Ignore this line and start at line 2
       from: 2,
     })
     const records: string[][] = []
-    this.whereaboutsApi.getVideoLinkEventsCSV(tokens, parser, day.startOf('day'), 1)
+    this.whereaboutsApi.getVideoLinkEventsCSV(tokens, parser, date, 31)
```

Comment:
will get up to 31 days of data

Reply:
If you use this function with a month that only has 28 days, will it get data from the 1/2/3 of next month? and is it fine if it does?

Reply:
Yes you are correct. But getting extra data will be ok because the end user will filter the data using specific dates